ovn-tester/ovn_workload.py

```diff
@@ -364,12 +363,9 @@ def start(self):
             self.central_nodes[0], sb_conn, inactivity_probe
         )
 
-        # ovn-ic configuration
-        self.icnbctl = ovn_utils.OvnIcNbctl(
-            None,
-            f'tcp:{self.cluster_cfg.node_net.ip + 2}:6645',
-            inactivity_probe,
-        )
+        # ovn-ic configuration: enable route learning/advertising to allow
+        # automatic pinging between cluster_net subnets in different AZs.
+        # This is allows IC connectivity checks.
```

Comment:
`This is allows` sounds odd.

Reply:
Done, I changed it to `This is required for`.